README.adoc

```diff
@@ -15,9 +15,7 @@ This `sonar-devon4j-plugin` provides a plugin extending https://sonarqube.org[So
 
 == Installation
 
-If you have https://sonarqube.org[SonarQube] installed, you only need to https://repo.maven.apache.org/maven2/com/devonfw/tools/sonar-devon4j-plugin/[download] and copy the latest version of this `sonar-devon4j-plugin` into the `extensions/plugins` folder of your SonarQube installation.
-Further you need to import the devonfw quality profiles (or activate all the rules of this plugin in your quality profile).
-For further details read the https://github.com/devonfw/sonar-devon4j-plugin/wiki/guide-sonar-qube-setup[SonarQube setup guide]
+If you have https://sonarqube.org[SonarQube] installed, you only need to https://repo.maven.apache.org/maven2/com/devonfw/tools/sonar-devon4j-plugin/[download] and copy the latest version of this `sonar-devon4j-plugin` into the `extensions/plugins` folder of your SonarQube installation. For further details read the https://github.com/devonfw/sonar-devon4j-plugin/wiki/guide-sonar-qube-setup[SonarQube setup guide]
```

Comment:
Shoundt we tell our users to install the plugin via marketplace?
(Sorry, for beeing off topic regarding the story of this PR).

Reply:
True, I will create a seperate PR for updating the readme.